implementations/rust/ockam/ockam_command/src/node/create.rs

```diff
@@ -165,5 +167,8 @@ async fn setup(ctx: Context, (c, cfg): (CreateCommand, OckamConfig)) -> anyhow::
     .await?;
     ctx.start_worker(NODEMAN_ADDR, node_man).await?;
 
+    ctx.start_worker("uppercase", Uppercase).await?;
```

Comment:
Not very important, but better place for this is NodeMan, since it's already responsible for starting services (Vault, Identity, Authenticated)

Reply:
Also, we'll be improving NodeMan persistence capabilities, so it will remember what it had started, to be able to spawn the same services during node restarts